extensions/antennas.sigmf-ext.md

```diff
@@ -0,0 +1,36 @@
+# Antenna Extension
+
+The `antenna` namespace extension defines one object for static antenna parameters extending `global` and one object for dynamic antenna parameters extending `annotations`.
+
+## Global
+
+### Antenna Object
+The `Antenna` object contains the following name/value pairs:
+
+|name|required|type|unit|description|
+|----|--------------|-------|-------|-----------|
+|`model`|true|string|N/A|Antenna make and model number. E.g. `"ARA CSB-16"`, `"L-com HG3512UP-NF"`.|
+|`type`|false|string|N/A|Antenna type. E.g. `"dipole"`, `"biconical"`, `"monopole"`, `"conical monopole"`.|
+|`low_frequency`|false|float|Hz|Low frequency of operational range.|
+|`high_frequency`|false|float|Hz|High frequency of operational range.|
+|`gain`|false|float|dBi|Antenna gain in direction of maximum radiation or reception.|
+|`horizontal_gain_pattern`|false|array of floats|dBi|Antenna gain pattern in horizontal plane from 0 to 359 degrees in 1 degree steps.|
```

Comment:
So, the SigMF spec, I think, in its current form, actually doesn't allow for arrays in the value datatype: https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#datatypes

But, I think this is a great example of why we should allow that. Further, looking at json.org, technically, all structures can be `values` in the JSON spec, which means the way we have this written currently kind of breaks that spec. I'm thinking we should add `object` and `array` to the table, which will fix the problem. I can add that change and push it to this branch when you are done making your changes.